crowdin.yml

```diff
@@ -0,0 +1,27 @@
+base_path: "."
+base_url: "https://neoforged.api.crowdin.com"
+
+# For interacting with Crowdin locally (via CLI), set these environment variables to their proper values
+project_id_env: "NEOFORGE_CROWDIN_PROJECT_ID"
+api_token_env: "NEOFORGE_CROWDIN_API_TOKEN"
+
+# Preserve directory structure (required by the use of the 'dest' field below)
+preserve_hierarchy: true
+
+# Prevent Crowdin-created commits from automatically triggering builds
+commit_message: "[ci skip]"
+# Automatically label PRs made by Crowdin
+pull_request_labels: [ "l10n" ]
+
+files:
+    -   source: "/src/main/resources/assets/neoforge/lang/en_us.json"
+        # Unfortunately, Crowdin doesn't have a placeholder which matches Minecraft's expected naming scheme: underscores
+        # with lowercase letters (since resource location are required to be fully lowercase).
+        #
+        # To make up for this unfortunate defect, **in the Crowdin interface**, we specify custom language mappings for
```

Comment:
You can add them here, like we do in mek. IIRC doing it in the crowdin UI will save them here anyway

Reply:
> IIRC doing it in the crowdin UI will save them here anyway

As I understand it, it depends on _where_ in the Crowdin UI you're performing these changes. If these language mappings are done as part of editing the configuration in the GitHub integration itself (<https://support.crowdin.com/enterprise/vcs-integrations-configuring-online/>), then it will modify the project configuration file as you say.

However, custom language mappings can also be specified in the same interface where languages are selected for the project (<https://support.crowdin.com/enterprise/project-settings/?q=languages#language-mapping>), which is how I specified them for the NeoForge project currently.

There are advantages and disadvantages to each approach (file-based vs project-based). I'll have to think about it more.

Reply:
The 2nd is for file export, which is not the vcs integration. One might think it affects both, but it likely doesn't

Reply:
As I understand it, both links I mentioned above refer to the same feature, just configured in different ways: one is via the project configuration file, and one is via the Crowdin UI.

Both features should affect the placeholders in the same way (though the project configuration file likely affects the specific source file(s) only, while the UI configuration affects all source files).